README.md

```diff
@@ -15,6 +15,10 @@ Below is a list of curated resources for Vuetify. If you have something that you
 ### Official
 - [Documentation](https://vuetifyjs.com)
 - [GitHub Repo](https://github.com/vuetifyjs/vuetify)
+- IDE Support
+  - [Visual Studio Code](https://github.com/vuetifyjs/vuetify-vscode)
```

Comment:
I can't remember the conversation but I believe we decided to let vetur handle this?